### PR TITLE
Ptrcheck no backtrace

### PR DESCRIPTION
**Pull request by Axel-Naumann**

### CODE REVIEW

File: core/metacling/src/TCling.cxx [hunk 1]
```diff
@@ -2442,7 +2442,7 @@ static int HandleInterpreterException(cling::MetaProcessor* metaProcessor,
    }
    catch (cling::InterpreterException& ex)
    {
-      Error("HandleInterpreterException", "%s.\n%s", ex.what(), "Execution of your code was aborted.");
+      Error("HandleInterpreterException", "%s\n%s", ex.what(), "Execution of your code was aborted.");
```
Comment (jalopezg-r00t) [inline core/metacling/src/TCling.cxx:2445]: Perhaps we should add a missing `.` in `Interpreter.cpp:1845` for projectroot.roottest.cling.specialobj.roottest_cling_specialobj_runf02 not to fail.

**Review by jalopezg-r00t — APPROVED** [review]
LGTM (but see comment that refers to the failing test).

File: core/metacling/src/TCling.cxx (hunk 1 above)
Comment (Axel-Naumann) [reply core/metacling/src/TCling.cxx:2445]: I'd rather update the ref file: adding a `.` after an expression printout seems "fragile", say `a+1..` is really hard to understand

**Comment by Axel-Naumann** [comment]
@phsft-bot build

**Pull request merged**